axelor-purchase/src/main/resources/views/PurchaseOrder.xml

```diff
@@ -579,6 +578,8 @@
     <action name="action-purchase-order-attrs-interco"/>
     <action name="action-purchase-order-attrs-external-reference"/>
     <action name="action-purchase-order-record-load-trading-name-config"/>
+    <action name="action-purchase-order-method-show-cancel-btn"
+      if="__config__.app.isApp('supplychain') &amp;&amp; statusSelect == 3"/>
```

Comment:
Please create new `action-group` in supplychain module calling `action-group-purchase-order-on-load-actions` and that new action